lib/emails/followerNotification.tsx

```diff
@@ -0,0 +1,174 @@
+import { Body, Head, Hr, Html, Img, Link, Preview, Text } from "@react-email/components";
+import { getBaseUrl } from "@/components/constants";
+
+type Props = {
+  eventType: "new_message" | "status_change" | "assignment_change" | "note_added";
+  triggeredByName: string;
+  conversationSubject: string;
+  customerEmail: string;
+  conversationLink: string;
+  eventDetails: {
+    message?: string;
+    oldStatus?: string;
+    newStatus?: string;
+    oldAssignee?: string;
+    newAssignee?: string;
+  };
+};
+
+const baseUrl = getBaseUrl();
+
+const getEventContent = ({ eventType, triggeredByName, eventDetails }: Props) => {
+  switch (eventType) {
+    case "new_message":
+      return {
+        title: "New message in conversation",
+        content: (
+          <>
+            <Text>
+              <strong>{triggeredByName}</strong> added a new message to the conversation.
+            </Text>
+            {eventDetails.message && (
+              <div
+                style={{
+                  background: "#f5f5f5",
+                  padding: "10px",
+                  borderLeft: "3px solid #007bff",
+                  margin: "12px 0",
+                  fontSize: "0.875rem",
+                }}
+              >
+                {eventDetails.message.substring(0, 200)}
+                {eventDetails.message.length > 200 ? "..." : ""}
+              </div>
+            )}
+          </>
+        ),
+      };
+    case "status_change":
+      return {
+        title: "Conversation status changed",
+        content: (
+          <Text>
+            <strong>{triggeredByName}</strong> changed the status: <span>{eventDetails.oldStatus}</span> →{" "}
+            <strong>{eventDetails.newStatus}</strong>
+          </Text>
+        ),
+      };
+    case "assignment_change":
+      return {
+        title: "Conversation assignment changed",
+        content: (
+          <Text>
+            <strong>{triggeredByName}</strong> changed the assignment:{" "}
+            <span>{eventDetails.oldAssignee || "Unassigned"}</span> →{" "}
+            <strong>{eventDetails.newAssignee || "Unassigned"}</strong>
+          </Text>
+        ),
+      };
+    case "note_added":
+      return {
+        title: "New note added",
+        content: (
+          <Text>
+            <strong>{triggeredByName}</strong> added a note to this conversation.
+          </Text>
+        ),
+      };
+    default:
+      return {
+        title: "Conversation updated",
+        content: (
+          <Text>
+            <strong>{triggeredByName}</strong> made changes to this conversation.
+          </Text>
+        ),
+      };
+  }
+};
+
+const FollowerNotificationEmail = (props: Props) => {
+  const { conversationSubject, customerEmail, conversationLink } = props;
+  const { title, content } = getEventContent(props);
+
+  return (
+    <Html>
+      <Head />
+      <Preview>
+        {title}: {conversationSubject}
```

Comment:
```suggestion
        {title}
```

We already have the conversation subject in the email subject